configs/localization/ssn/README.md

```diff
@@ -37,12 +37,34 @@ You can use the following command to train a model.
 python tools/train.py ${CONFIG_FILE} [optional arguments]
 ```
 
-Example: train SSN model on thumos14 dataset.
+Example: train SSN on THUMOS14 dataset.
 
 ```shell
 python tools/train.py configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py
 ```
 
+Example: train SSN on ActivityNet-1.3 dataset.
+
+1. Get the result of action proposals. (Take proposal generator BMN as an example.)
+
+```shell
+python tools/test.py configs/localization/bmn/bmn_400x100_2x8_9e_activitynet_feature.py checkpoints/SOME_CHECKPOINT.pth --eval AR@AN --out results.json
+```
+
+2. Convert the output proposal file of proposal generator (BSN, BMN) into the input proposal file of action classifier. (Currently supports SSN and P-GCN, not including TSN, I3D etc.)
+
+```shell
+python tools/data/activitynet/convert_proposal_format.py
+```
+
+3. train SSN model.
+
+Notes: There is a large gap between THUMOS14 and ActivityNet-1.3, therefore hyperparameters in config need to be carefully tuned.
```

Comment:
Do you have "carefully tuned" config for THUMOS14?

Reply:
Parameters of SSN on THUMOS14 follow the setting of MMAction1, while MMAction1 does not provide the setting of SSN for ActivityNet.

Reply:
In L65, the config file is called `ssn_r50_1000e_anet_rgb_train.py`.
Is this new config designed for anet, or `thumos14`?